mindsdb/integrations/handlers/auto_ts_handler/auto_ts_handler.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+from auto_ts import auto_timeseries as ATS
+from typing import Optional
+import dill
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class Auto_ts_Handler(BaseMLEngine):
+    """
+    Auto_ts handler class
+    """
+    name = 'auto_ts'
+
+    @staticmethod
+    def create_validation(self, args):
+        """
+        Create validation set from training set
+        """
+        score_type = ['rmse', 'normalized_rmse']
+        frequency = ['B', 'C', 'D', 'W', 'M', 'SM', 'BM', 'CBM', 'MS',
+                     'SMS', 'BMS', 'CBMS', 'Q', 'BQ', 'QS', 'BQS',
+                     'A,Y', 'BA,BY', 'AS,YS', 'BAS,BYS', 'BH',
+                     'H', 'T,min', 'S', 'L,ms', 'U,us', 'N']
+        model = ['best', 'prophet', 'stats', 'ARIMA', 'SARIMAX', 'VAR', 'ML']
+
+        if 'using' in args:
+            args = args['using']
+
+        if 'score_type' in args and args['score_type'] not in score_type:
+            raise Exception(f"score_type must be one of {score_type}")
+
+        if 'time_interval' in args and args['time_interval'] not in frequency:
+            raise Exception(f"frequency must be one of {frequency}")
+
+        if 'non_seasonal_pdq' in args and not (isinstance(args['non_seasonal_pdq'], tuple) or args['non_seasonal_pdq'] == 'None'):
+            raise Exception("non_seasonal_pdq must be a tuple")
+
+        if 'model_type' in args and args['model_type'] not in model:
+            raise Exception(f"model_type must be one of {model}")
+
+        if 'ts_column' not in args:
+            raise Exception("Column containing time series must be defined")
+
+        if 'target' not in args:
+            raise Exception("target must be defined")
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[dict] = None) -> None:
+        """
+        Creates Auto_ts model using the input df.
+        """
+        args = args['using']
```

Comment:
This dict will contain a `timeseries_settings` key when created using time series syntax.